docs/source/recipes/lazyload.md

```diff
@@ -35,7 +35,7 @@ You can find the complete `@loadable/component` documentation here: https://load
 You can check the code splitting state by using the included bundle analyzer:
 
 ```bash
```

Comment:
We want to replace all instances of the bash language with shell so that we are shell-neutral.

````suggestion
```shell
````